registry/Dockerfile

```diff
@@ -28,7 +28,7 @@ COPY parser.go $DISTRIBUTION_DIR/configuration/parser.go
 # This step is required to generate build config
 FROM base AS version
 ARG PKG="github.com/docker/distribution"
-ENV REGISTRY_VERSION 2.8.2
+ENV REGISTRY_VERSION 2.8.3
```

Comment:
registry service is still broken you can release image in ap-vendor but lets not point in astronomer/astronomer